packages/medusa/src/scripts/migrate-to-pricing-module.ts

```
@@ -0,0 +1,232 @@
+import { CurrencyService, PriceListService } from "../services"
+import { IPricingModuleService, PricingTypes } from "@medusajs/types"
+
+import { AwilixContainer } from "awilix"
+import { PriceList } from "../models"
+import { createDefaultRuleTypes } from "./utils/create-default-rule-types"
+import dotenv from "dotenv"
+import express from "express"
+import loaders from "../loaders"
+import { migrateProductVariantPricing } from "./utils/migrate-money-amounts-to-pricing-module"
+
+dotenv.config()
+
+const BATCH_SIZE = 10
+
+const migratePriceLists = async (container: AwilixContainer) => {
+  const pricingModuleService: IPricingModuleService = container.resolve(
+    "pricingModuleService"
+  )
+  let offset = 0
+
+  const priceListCoreService: PriceListService =
```

suggestion: should we throw if the module is not present just to alter the user to install and configure it? or do we assume that if he is running it then he configured it

---

We're running the loaders at the beginning of the script, this should always be present in that case.